### PR TITLE
[CHAOS-45] Add Documentation to the ChaosCLI

### PR DESCRIPTION
**Pull request by Azoam**
## What does this PR do?

- [ ] Adds new functionality
- [ ] Alters existing functionality
- [ ] Fixes a bug
- [X] Improves Documentation

A brief description of changes to implementation or controller behavior:
Adds simple Documentation for the ChaosCLI.

### CODE REVIEW

File: cli/chaosli/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# ChaosCLI
```
Comment (ptnapoleon) [inline cli/chaosli/README.md:1]: This should be `Chaosli` if that's the name we're going with. Or title this `Chaos Controller CLI` and then the first paragaph would be, `The Chaos Controller CLI, chaosli, ....`

File: cli/chaosli/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+The ChaosCLI is motivated by creating a more user friendly and digestible chaos-controller experience. The CLI pairs with your Disruption configuration, giving you information to give you a better sense of what is going on or what will happen when you apply new disruptions. With features like explain and validation, it gives users, not only a better understanding, but a better experience with the controller.
```
Comment (ptnapoleon) [inline cli/chaosli/README.md:3]:
```suggestion
The Chaos Controller CLI, `chaosli`, is meant to help create a more user friendly and digestible chaos-controller experience. The CLI pairs with your Disruption's definition, giving you information to give you a better sense of what is going on, and what will happen when you apply new disruptions. With features like explain and validation, it gives users, not only a better understanding, but a better experience with the controller.  
```

File: cli/chaosli/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+---
+- Validate
+- Explain
+- Create
+
+#### Validate
+---
+Usage: `go run chaosli/main.go validate --path <path to disruption file>`
+
+Description: Validates your disruption files to make sure everything is correctly formatted.
```
Comment (ptnapoleon) [inline cli/chaosli/README.md:16]:
```suggestion
Description: Validates your disruption file to make sure everything is correctly formatted.
```

File: docs/faq.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
 # FAQ
 
+## Is there any specific tooling that can help me create/understand my disruptions?
+
+Yes! Take a look at [Chaosli](../cli/chaosli/README.md). This is a command line interface that has many features which include:
+explaining your disruption configuration is a human digestible way.
```
Comment (ptnapoleon) [inline docs/faq.md:6]: Take a look at the preview, this doesn't line break in the way I think you expect?

File: cli/chaosli/README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+$ go run chaosli/main.go validate --path=../examples/network_delay.yaml
+Error: cannot apply an empty disruption - at least one of Network, DNS, DiskPressure, NodeFailure, CPUPressure fields is needed
+```
+
+#### Explain
+---
+Usage: `go run chaosli/main.go explain --path <path to disruption file>`
+
+Description: Prints out a summary of the disruption defined by `path`.
```
Comment (ptnapoleon) [inline cli/chaosli/README.md:33]: I think we should try to have consistency across `validate` and `explain` with how we refer to `path` in the description

File: cli/chaosli/README.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+💉 injects a network disruption ...
+	💥 applies network failures on outgoing traffic.
+		💣 applies a packet delay of 1000 ms.
+			💣 applies a jitter of 1000 ms to the delay value to add randomness to the delay.
+=======================================================================================================================================
+```
+
+#### Creation
+---
+Usage: `go run chaosli/main.go create --path <path of new disruption file>`
```
Comment (ptnapoleon) [inline cli/chaosli/README.md:57]:
```suggestion
Usage: `go run chaosli/main.go create --path <path to output generated disruption file>`
```

File: cli/chaosli/README.md (hunk 5 above)
Comment (Azoam) [reply cli/chaosli/README.md:33]: Pushing a new commit, let me know if my change isnt what you meant.

**Review by ptnapoleon — APPROVED** [review]

File: docs/faq.md [hunk 7]
```diff
@@ -1,5 +1,15 @@
...
 
+## Is there any specific tooling that can help me create/understand my disruptions?
+
+Yes! Take a look at [Chaosli](../cli/chaosli/README.md). This is a command line interface that has many features which include:
+
+Explaining your disruption configuration is a human digestible way.
+
+Creating new disruptions from scratch answering simple questions.
+
+Validating your disruptions before running them.
```
Comment (takakonishimura) [inline docs/faq.md:11]: bullets for these 3 lines? :D

**Pull request merged**